Cargo.toml

```diff
@@ -4,6 +4,7 @@ members = [
     "argon2",
     "balloon-hash",
     "bcrypt-pbkdf",
+    "benches",
```

Comment:
No, keep the `benches` crate outside of the common workspace. Otherwise its dependencies become part of the workspace which we do not want. You may need to add `**/Cargo.lock` to `.gitignore` to prevent `bencehs/Cargo.lock` from being committed.